frontend/fingertips-frontend/components/molecules/ArrowExpander/index.tsx

```diff
@@ -47,24 +47,25 @@ interface ExpanderProps {
   openTitle?: string;
   closeTitle?: string;
   open?: boolean;
+  toggleClickFunction: () => void;
 }
 export const ArrowExpander = ({
   children,
   fill = '#1D70B8',
   openTitle,
   closeTitle,
```

Comment:
```suggestion
  openTitle = 'Open',
  closeTitle = 'Close',
```
some defaults for the optional props?